pkg/user/apis/user/v1/conversion.go

```diff
@@ -8,22 +8,11 @@ import (
 )
 
 func addConversionFuncs(scheme *runtime.Scheme) error {
-	if err := scheme.AddFieldLabelConversionFunc("v1", "Group",
-		oapi.GetFieldLabelConversionFunc(userapi.GroupToSelectableFields(&userapi.Group{}), nil),
-	); err != nil {
-		return err
-	}
-
 	if err := scheme.AddFieldLabelConversionFunc("v1", "Identity",
 		oapi.GetFieldLabelConversionFunc(userapi.IdentityToSelectableFields(&userapi.Identity{}), nil),
 	); err != nil {
 		return err
 	}
 
-	if err := scheme.AddFieldLabelConversionFunc("v1", "User",
-		oapi.GetFieldLabelConversionFunc(userapi.UserToSelectableFields(&userapi.User{}), nil),
-	); err != nil {
-		return err
-	}
```

Comment:
Why is this required [for buildconfigs](https://github.com/openshift/origin/pull/15963/files#diff-c6033988686cf6611cf7e928d8dbea2aR178) but not here?